src/components/community/MattDecuir.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import faGithub from '@fortawesome/fontawesome-free-brands/faGithub';
+import faLinkedin from '@fortawesome/fontawesome-free-brands/faLinkedin';
+import faTwitter from '@fortawesome/fontawesome-free-brands/faTwitter';
+import faLink from '@fortawesome/fontawesome-free-solid/faLink';
+
+const MattDecuir = () => {
```

Comment:
At some point, we may wish to make these follow a particular format, like a component with props:

```
type Props = {
  name: String,
  bio: String,
  twitter: ?String,
  linkedIn: ?String,
  website: ?String
]
```

Reply:
apologies for the pseudo-flow type defs :)

Reply:
Ohhhhhh I think I see where you're going with this -- like if we just have bio components for each community member, we could just pull an index of that folder and use that as the `communityMembers` data structure, instead of having them be weirdly referencing and dependent on each other?